supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -175,6 +175,8 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor,           // W
     |"
     |cat ${dockerStderr}
     |echo "--${boundary}--"
+    |sleep 2 # Provide sidecar opportunity to delocalize - not sure this is needed
```

Comment:
oh, and no, there's not really a maximum. I'm told that delocalization might involve huge files that will take a while to copy out to S3. I'll be syncing as we go, but if a large file write happens at the end of the process, it could take a while to complete the S3 sync. The delay is so that the sidecar can detect that the job is done, denoted by the RC file being written (detected by ionotify) and pause the task container while it finishes the copy. I just want to make sure the task container doesn't exit before the sidecar has a chance to pause it. I don't know the ionotify latency time or whether the sync call waits for ionotify calls to be delivered before returning.